doc/api/fs.md

```diff
@@ -1549,6 +1549,23 @@ Synchronous version of [`fs.readFile`][]. Returns the contents of the `file`.
 If the `encoding` option is specified then this function returns a
 string. Otherwise it returns a buffer.
 
+_Note: The behavior of `fs.readFileSync()` is platform specific. As such,
```

Comment:
Won't the example clear this up? I agree your comment give room for less confusion
But the line below would clear up that confusion: 

When the given path identifies a directory, the behavior of `fs.readFile()` and `fs.readFileSync()` is platform specific